### PR TITLE
stale: migrate from bot to action

### PR DESCRIPTION
**Pull request by ericLemanissier**
The bot has been deprecated for some time: https://github.com/probot/stale

### CODE REVIEW

**Review by CJCombrink — APPROVED** [review]

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '34 6 * * *'
```
[the remark this answers is not in the document]
Comment (ericLemanissier) [reply .github/workflows/stale.yml:10]: this time is generated randomly by github when creating a workflow via web interface like https://github.com/conan-io/conan-center-index/new/master?filename=.github%2Fworkflows%2Fstale.yml&workflow_template=automation%2Fstale
This avoids having all the scheduled actions of the world starting simultaneously.

**Review by uilianries — COMMENTED** [review]
Looking good. I have this file running in another repository. I'll wait until tomorrow to check if it closes everything and does not touch issues. So will be good to go.

**Comment by ericLemanissier** [comment]
@uilianries arr we good to go?

**Review by uilianries — APPROVED** [review]
LGTM. After testing during the weekend in a private repository, the result is good. No issues touched, only PRs and marking/closing according to the date.

**Review by RubenRBS — APPROVED** [review]

**Pull request merged**